ros_ws/src/car_control/src/drive_parameters_multiplexer.cpp

```diff
@@ -0,0 +1,53 @@
+#include "drive_parameters_multiplexer.h"
+
+DriveParametersMultiplexer::DriveParametersMultiplexer()
+{
+    this->m_drive_parameters_publisher = this->m_node_handle.advertise<drive_msgs::drive_param>(TOPIC_DRIVE_PARAM, 1);
+    this->m_last_updated_source = NULL;
+    auto callback =
+        std::bind(&DriveParametersMultiplexer::onUpdate, this, std::placeholders::_1, std::placeholders::_2);
+    this->m_sources.push_back(
```

Comment:
I figured that it's not worth the effort since only one more source will be added and there is no configuration needed when working with the car.
Furthermore, there is no reason to change them later and using a config file saves little time compared to changing the code.
If you would like to have a config file anyway, I'll look into it.